lang/pl.json

```
@@ -1432,5 +1433,122 @@
+  "CoC7.ChaosiumCanvasInterface.DrawingToggle.TriggerButton.Hint": "Jeśli przycisk myszy jest ustawiony na 'Oba Przyciski', ten przycisk jest używany do wlączania strefy",
+  "CoC7.ChaosiumCanvasInterface.DrawingToggle.TriggerRegionUuids.Title": "Włącz tę strefę",
+  "CoC7.ChaosiumCanvasInterface.DrawingToggle.TriggerRegionUuids.Hint": "",
+  "CoC7.ChaosiumCanvasInterface.DrawingToggle.TriggerAsButton.Title": "Przyciskiem myszy",
+  "CoC7.ChaosiumCanvasInterface.DrawingToggle.TriggerAsButton.Hint": "",
+  "CoC7.ChaosiumCanvasInterface.MapPinToggle.Button.Title": "Przycisk myszy",
+  "CoC7.ChaosiumCanvasInterface.MapPinToggle.Button.Hint": "Który przycisk myszy ma włączać to zachowanie",
+  "CoC7.ChaosiumCanvasInterface.MapPinToggle.Action.Title": "Akcja",
+  "CoC7.ChaosiumCanvasInterface.MapPinToggle.Action.Hint": "Czy włączyć, wyłączyć albo przełączyć widok pinezek na mapie i dokumentów",
+  "CoC7.ChaosiumCanvasInterface.MapPinToggle.Document.Title": "Wybierz dokumenty",
+  "CoC7.ChaosiumCanvasInterface.MapPinToggle.Document.Hint": "Ustaw uprawnienia dokumentów",
+  "CoC7.ChaosiumCanvasInterface.MapPinToggle.Note.Title": "Notatki sceny",
+  "CoC7.ChaosiumCanvasInterface.MapPinToggle.Note.Hint": "Ustaw teksturę pinezek dla notatek sceny",
+  "CoC7.ChaosiumCanvasInterface.MapPinToggle.PermissionShow.Title": "Uprawnienia pokazywania",
+  "CoC7.ChaosiumCanvasInterface.MapPinToggle.PermissionShow.Hint": "Gdy ustawione jest pokazywanie, ustaw uprawnienia dokumentów na ten poziom",
+  "CoC7.ChaosiumCanvasInterface.MapPinToggle.PermissionHide.Title": "Uprawnienia ukrywania",
+  "CoC7.ChaosiumCanvasInterface.MapPinToggle.PermissionHide.Hint": "Gdy ustawione jest ukrywanie, ustaw uprawnienia dokumentów na ten poziom",
+  "CoC7.ChaosiumCanvasInterface.OpenDocument.Button.Title": "Przycisk myszy",
+  "CoC7.ChaosiumCanvasInterface.OpenDocument.Button.Hint": "Który przycisk myszy ma włączać to zachowanie",
+  "CoC7.ChaosiumCanvasInterface.OpenDocument.Permission.Title": "Kto może kliknąć",
+  "CoC7.ChaosiumCanvasInterface.OpenDocument.Permission.Hint": "",
+  "CoC7.ChaosiumCanvasInterface.OpenDocument.Document.Title": "Wybierz dokument",
+  "CoC7.ChaosiumCanvasInterface.OpenDocument.Document.Hint": "Który dokument / strona dokumentu / aktor",
+  "CoC7.ChaosiumCanvasInterface.OpenDocument.Anchor.Title": "Opcjonalna sekcja",
+  "CoC7.ChaosiumCanvasInterface.OpenDocument.Anchor.Hint": "Jeśli ustawiona jest strona dokumentu, pozwala skoczyć do konkretnej sekcji",
+  "CoC7.ChaosiumCanvasInterface.Permission.Always": "Zawsze",
+  "CoC7.ChaosiumCanvasInterface.Permission.Document": "Może widzieć dokument",
+  "CoC7.ChaosiumCanvasInterface.Permission.GM": "Strażnik",
+  "CoC7.ChaosiumCanvasInterface.Permission.SeeTile": "Może widzieć tytuł",
+  "CoC7.ChaosiumCanvasInterface.PlaySound.Button.Title": "Przycisk myszy",
+  "CoC7.ChaosiumCanvasInterface.PlaySound.Button.Hint": "Który przycisk myszy ma włączać to zachowanie",
```

```suggestion
  "CoC7.ChaosiumCanvasInterface.PlaySound.Button.Hint": "Który przycisk myszy ma wywołać to zachowanie",
```
Usually triggers are activated instead of turned on (to perform an action):
"włączać" - to enable
"wywołać" - to trigger

---

I think both are good in this case, but "wywołać" sounds better here. Changing here and in other places.